tokio-util/src/task/spawn_pinned.rs

```diff
@@ -9,7 +9,57 @@ use tokio::sync::mpsc::{unbounded_channel, UnboundedReceiver, UnboundedSender};
 use tokio::sync::oneshot;
 use tokio::task::{spawn_local, JoinHandle, LocalSet};
 
-/// A handle to a local pool, used for spawning `!Send` tasks.
+#[derive(Debug)]
+pub struct WorkerIdxError(usize, usize);
+
+impl fmt::Display for WorkerIdxError {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(
+            f,
+            "Index {} out of bounds, only {} workers in pool",
+            self.0, self.1
+        )
+    }
+}
```

Comment:
This should implement the `Error` trait. Also, would probably make sense to name the fields so the `Debug` printing is more helpful.